financial-qa-search/QAReranker/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM jinaai/jina
```

Comment:
Here you would do the downloading of the model so that it is available inside the image